modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -5940,73 +5940,12 @@ public void testPutCustomObjectEntryUnlinkNestedCustomObjectEntriesByExternalRef
 	public void testPutCustomObjectEntryWithDuplicateExternalReferenceCode()
```

Comment:
If there is a test for the PUT and another one for the PATCH when the external reference code is duplicated, I miss a test for updating the ERC in the PUT endpoint

testPatchCustomObjectEntryExternalReferenceCode vs testPutCustomObjectEntryExternalReferenceCode

Could we include the test in this PR?

Reply:
Sure!!